src/Pact/Types/Hash.hs

```diff
@@ -9,9 +9,6 @@ module Pact.Types.Hash
 
   -- Integer hashing
   , hashAsBasedInteger
-  , decimalStringToInteger
```

Comment:
Where is `hashAsBasedInteger` used?

Reply:
I should've labeled this WIP - `hashAsBasedInteger` is the main way we convert between bases

Reply:
There's no reason for these to be in `Types/Hash.hs` anymore, go ahead and move them next to where they're used in `Pact/Native.hs` and revert all changes to this file.

Reply:
Done.